src/formats/taglist.c

```diff
@@ -81,7 +81,7 @@ const sd_tag sd_taglist[] = {
     {"mc",  0, NULL                                                                                                   },
     {"md",  1, "Destroy animation N"                                                                                  },
     {"mg",  1, "Gravity for spawned animation, default 0"                                                             },
-    {"mi",  1, NULL                                                                                                   },
+    {"mi",  1, "Instance count for spawned animation, default 1"                                                      },
```

Comment:
The taglist is autogenerated from pyomftools, would be nice to get this to the csv file in there too.

Reply:
(or if it's in the wiki, I will add it from there eventually)